app/controllers/users.py

```diff
@@ -1184,8 +1199,20 @@ def post(self, user_id):
             log_activity('User', status='Success',
                          operation='Enable',
                          description='Enabled user Successfully',
-                         a_project_id=project.id,
-                         a_cluster_id=project.cluster_id)
+                         a_user_id=user.id
+                         )
+            html_layout = render_template(
+                'user/user_disable_enable.html',
+                email=user.email,
+                name=user.name,
+                status='enabled')
+            send_email(
+                'lanternnassi@gmail.com',
```

Comment:
Remove the hard coded email
```suggestion
                user.email,
```